### PR TITLE
[Flink]paimon-action support normal json format

### LINKED ISSUE #3306: [Feature] paimon-action supports normal json format?
### Search before asking

- [X] I searched in the [issues](https://github.com/apache/paimon/issues) and found nothing similar.

### Motivation

paimon-action currently only supports data warehousing in cdc format. Does not seem to support normal json format? In a log warehousing scenario, the business often adds new fields, causing flink to do the corresponding field restart task. If paimon supports the common json format, the whole warehousing will be more smooth

### Solution

example:

### Anything else?

_No response_

### Are you willing to submit a PR?

- [X] I'm willing to submit a PR!

### PR DESCRIPTION
**Pull request by MOBIN-F**
### Purpose

Linked issue: close #3306

### CODE REVIEW

**Comment by MOBIN-F** [comment]
@JingsongLi  please help with pr review, tks

**Review by JingsongLi — COMMENTED** [review]
Can you write documentation too?

**Comment by MOBIN-F** [comment]
I added some documentation, please help review the code @JingsongLi

File: paimon-flink/paimon-flink-cdc/src/test/java/org/apache/paimon/flink/action/cdc/kafka/KafkaJsonSyncTableActionITCase.java [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+/** IT cases for {@link KafkaSyncTableAction}. */
+public class KafkaJsonSyncTableActionITCase extends KafkaSyncTableActionITCase {
+
+    @Test
+    @Timeout(60)
+    public void testSchemaEvolution() throws Exception {
+        String topic = "schema-evolution";
+        Map<String, String> tableOptions = new HashMap<>();
+        tableOptions.put(CoreOptions.BUCKET.key(), "-1");
```
Comment (JingsongLi) [inline paimon-flink/paimon-flink-cdc/src/test/java/org/apache/paimon/flink/action/cdc/kafka/KafkaJsonSyncTableActionITCase.java:45]: no config, use default value?

File: paimon-flink/paimon-flink-cdc/src/test/java/org/apache/paimon/flink/action/cdc/kafka/KafkaJsonSyncTableActionITCase.java [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+                rowType,
+                Collections.emptyList());
+    }
+
+    @Test
+    @Timeout(60)
+    public void testComputedColumn() throws Exception {
+        String topic = "computed_column";
+        Map<String, String> tableOptions = new HashMap<>();
+        tableOptions.put(CoreOptions.BUCKET.key(), "-1");
```
Comment (JingsongLi) [inline paimon-flink/paimon-flink-cdc/src/test/java/org/apache/paimon/flink/action/cdc/kafka/KafkaJsonSyncTableActionITCase.java:101]: no config, use default value?

File: docs/content/flink/cdc-ingestion/pulsar-cdc.md [hunk 3]
```diff
@@ -156,6 +160,28 @@ Then you can submit synchronization job:
...
+    --table test_table \
+    --partition_keys pt \
+    --computed_column 'pt=date_format(event_tm, yyyyMMdd)' \
+    --kafka_conf properties.bootstrap.servers=127.0.0.1:9020 \
+    --kafka_conf topic=test_log \
+    --kafka_conf properties.group.id=123456 \
+    --kafka_conf value.format=json \
+    --catalog_conf metastore=hive \
+    --catalog_conf uri=thrift://hive-metastore:9083 \
+    --table_conf bucket=4 \
```
Comment (JingsongLi) [inline docs/content/flink/cdc-ingestion/pulsar-cdc.md:181]: really work? you should no config, use default value.

File: docs/content/flink/cdc-ingestion/kafka-cdc.md [hunk 4]
```diff
@@ -157,6 +161,28 @@ Then you can submit synchronization job:
...
+    --table test_table \
+    --partition_keys pt \
+    --computed_column 'pt=date_format(event_tm, yyyyMMdd)' \
+    --kafka_conf properties.bootstrap.servers=127.0.0.1:9020 \
+    --kafka_conf topic=test_log \
+    --kafka_conf properties.group.id=123456 \
+    --kafka_conf value.format=json \
+    --catalog_conf metastore=hive \
+    --catalog_conf uri=thrift://hive-metastore:9083 \
+    --table_conf bucket=4 \
```
Comment (JingsongLi) [inline docs/content/flink/cdc-ingestion/kafka-cdc.md:182]: really work? you should no config, use default value.

File: docs/content/flink/cdc-ingestion/pulsar-cdc.md (hunk 3 above)
Comment (MOBIN-F) [reply docs/content/flink/cdc-ingestion/pulsar-cdc.md]: sorry, my oversight

**Review by JingsongLi — APPROVED** [review]
+1

**Comment by JingsongLi** [comment]
Thanks @MOBIN-F ! Merging....

**Pull request merged**